ts/screens/onboarding/OnboardingNotificationsPreferencesScreen.tsx

```diff
@@ -104,15 +104,15 @@ const Header = memo(({ isFirstOnboarding }: { isFirstOnboarding: boolean }) => (
     <H1 color={isFirstOnboarding ? "bluegreyDark" : "white"}>
       {I18n.t(
         isFirstOnboarding
-          ? "profile.preferences.notifications.titleNewUser"
-          : "profile.preferences.notifications.title"
+          ? "profile.preferences.notifications.title"
+          : "profile.preferences.notifications.titleExistingUser"
```

Comment:
Just a suggestion: maybe could be easier to read the code if we avoid the repetition for `isFirstOnboarding` ternary check and creating in single point an object shaped something like `{ title, subtitle }` and using it in the component.

something like:

```typescript
const Header = memo(({ isFirstOnboarding }: { isFirstOnboarding: boolean }) => {
  const { title, subtitle } = isFirstOnboarding ? {
      title: I18n.t("profile.preferences.notifications.title"),
      subtitle: I18n.t("profile.preferences.notifications.subtitle")
    } : {
      title: I18n.t("profile.preferences.notifications.titleExistingUser"),
      subtitle: I18n.t("profile.preferences.notifications.subtitleExistingUser")
    };

  return (
    <View style={styles.contentHeader}>
      <H1 color={isFirstOnboarding ? "bluegreyDark" : "white"}>
        {title}
    </H1>
    <Body color={isFirstOnboarding ? "bluegreyDark" : "white"}>
      {subtitle}
    </Body>
  </View>
)}
```

what do you think?

Reply:
changed https://github.com/pagopa/io-app/pull/4139/commits/8d5931a24357f1b9eb847d06f4ea2ddee390e1a6